README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+![FlowChart](Flowchart.png)
+```
+SecretCode ← Generated Random Word;
+UserGuess ←  User's Selected Guess;
+
+If (SecretCode == UserGuess)
+{
+	Output << Correct;
+	Output << Terminal Unlocked;
+	Unlock Terminal;
+	End Program;
+}
+Else
+{
+	numberCorrect ← 0;
+	numberOfLetters ← 0;
+	For each letter in UserGuess { 				► i = letter number
+		if (SecretCode[i] == UserGuess[i])
+		{
+			numberCorrect ++;
+		}
+		numberOfLetters ++;
```

Comment:
So `numberOfLetters` is used to calculate the length of the string -- could say this explicitly